### DISCUSSION TITLE
No Copy/Paste Between Projects?

### DISCUSSION THREAD

**Discussion by cirquesbreeze** [keiwando/toonsquid#171] [Q&A]
How do I copy stuff between two different projects? Copying and pasting seems to only work for layers within a project.

I'd like to copy different vector objects I've drawn (in ToonSquid) between two or more projects. This way I can keep different assets handy in different projects. 

Otherwise I need to cram ALL assets into one ToonSquid project. Example: if I make 1,000 animated videos alone this year, that's a lot of stuff (backrounds, characters, props, etc.) in one project. 

Another option: make Library and created Symbols accessible across projects globally in the app. I also don't understand why Library and Symbols only work on a per-project basis.

**Comment by Cat-Butler** [comment]
While ToonSquid does have some vector functionality, on the whole it is mostly a pixel/raster app (only Keiwan knows if this will change in the future), and does not currently have vector export/import functionality.
Library content and symbols import/export well as image/movie format (I've tried it for a mouth set).

One way to work around this would be to create/maintain your vector props and content in an external app (personally I'd recommend Affinity Design and its Asset functionality, or maybe something similar but simpler in Amadine), and import appropriate content as images in the appropriate resolution for the current scene. 

I've read in various forums that this approach has even been followed by a couple of large animation studios in the past for desktop animation, at least for a couple projects discussed (the advantage of generating assets in different resolutions was one discussion topic, including for importing into both animation and compositing/video editing apps, but it was a few years back).

**Comment by jordanwmartin** [comment]
This is my biggest issue with using ToonSquid professionally. Everything else about ToonSquid is amazing. It has almost everything I need and want in an animation application, but if I can't share assets between projects (and in the best possible world, import assets from other applications as svg or some other standard format), then it might be a nonstarter.
